src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -306,6 +306,26 @@ type Object struct {
 
 	// Prealloc enables memory preallocation
 	Prealloc bool
+
+	// SevPolicy is the policy for the SEV instance. For more info, see AMD document 55766
+	// This is only relevant for sev-guest objects
+	SevPolicy uint32
+
+	// SnpPolicy is the policy for the SNP instance. For more info, see AMD document 56860
+	// This is only relevant for sev-snp-guest objects
+	SnpPolicy uint64
```

Comment:
When we first implemented this I wanted to revisit having separate variables for the SEV Policy and the SNP Policy. Would it be better to just have one? I guess with two it is easier to setup SEV and SNP on the same node and switch between them. That probably isn't a very common use case, though.